src/hooks/use-maps-instant.ts

```
@@ -0,0 +1,90 @@
+import { useSession } from '@/contexts/session-instantdb'
+import { Map } from '@/interfaces'
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)